pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1812,14 +1812,17 @@ type NFSGaneshaSpec struct {
 
 // GaneshaRADOSSpec represents the specification of a Ganesha RADOS object
 type GaneshaRADOSSpec struct {
-	// Pool used to represent the Ganesha's pool name in version older than 16.2.7
-	// As of Ceph Pacific 16.2.7, NFS Ganesha's pool name is hardcoded to ".nfs", so this
-	// setting will be ignored.
+	// Pool used store the shared configuration for NFS-Ganesha daemons.
+	// This setting is required for Ceph v15 and ignored for Ceph v16.
+	// As of Ceph Pacific 16.2.7, NFS Ganesha's pool name is hardcoded to ".nfs".
 	// +optional
 	Pool string `json:"pool,omitempty"`
 
-	// Namespace is the RADOS namespace where NFS client recovery data is stored.
-	Namespace string `json:"namespace"`
+	// Namespace in the pool where shared NFS-Ganesha config is stored.
+	// This setting is required for Ceph v15 and ignored for Ceph v16.
+	// As of Ceph Pacific 16.2.7, NFS Ganesha's pool name is hardcoded to the name of the CephNFS.
```

Comment:
It's the namespace, not the pool name, right?
```suggestion
	// As of Ceph Pacific 16.2.7, the namespace is internally set to the name of the CephNFS.
```

Reply:
Correct. The namespace of the pool. Should I make that more clear?